draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1082,59 +1110,51 @@ document does not discuss inversion algorithms.
 
 Encodings are closely related to mappings.
 Like a mapping, an encoding is a function that outputs a point on an elliptic curve.
-In contrast to a mapping, however, the input to an encoding is an arbitrary string.
-Encodings can be deterministic or probabilistic.
+In contrast to a mapping, however, the input to an encoding is an arbitrary-length
+byte string. Encodings can be deterministic or probabilistic.
 Deterministic encodings are preferred for security, because probabilistic
-ones are more likely to leak information through side channels.
+ones are more likely to leak information through side channels. See {{VR20}}
+as an example of this type of leakage leading to a security vulnerability.
 
-This document constructs deterministic encodings by composing a hash function H
+This document constructs deterministic encodings by composing a hash function H'
```

Comment:
As an alternative, I was going to suggest `Hf`. Maybe slightly easier to read than H'?

Reply:
That'll work too! Want to make that change in your pass after rebasing?